### ISSUE TITLE
Autorefill triggered

### ISSUE THREAD

**Issue by ghost** [AmericanCoinAMC/contract#1]
Is there a way for us to find out which transactions made use of the autorefill functionality? And also get the sell price?

**Comment by mangoklie** [comment]
Yes, the fuction sell emits a Transfer Event and the API detects this Transaction and stores it in Firebase DB. 
The addresses involved are the contract's address and the account using the autorefill.
